Buildings/Controls/OBC/ASHRAE/G36_PR1/Generic/SetPoints/GroupStatus.mo

```diff
@@ -151,51 +156,119 @@ block GroupStatus "Block that outputs the zone group status"
     annotation (Placement(transformation(extent={{100,-320},{140,-280}}),
       iconTransformation(extent={{100,-210},{140,-170}})));
 
+  CDL.Routing.BooleanArrayFilter zonOccFil(
```

Comment:
It would a good practice to use the full path, `Buildings.Controls.OBC.CDL....`. Also, the parameters can be finalized:
```
final nin=numZon,
final nout=numZonGro,
final msk=zonGroMsk
```